mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.java

```diff
@@ -514,6 +529,20 @@ protected static int determineTransition(FragmentUpdateReason reason) {
         }
     }
 
+    public static class SitemapLoadFailureFragment extends StatusFragment {
+        public static SitemapLoadFailureFragment newInstance(CharSequence message) {
+            SitemapLoadFailureFragment f = new SitemapLoadFailureFragment();
+            f.setArguments(buildArgs(message, R.drawable.ic_openhab_appicon_24dp /* FIXME */,
```

Comment:
Two ideas for an icon:
https://github.com/Templarian/MaterialDesign/blob/master/icons/svg/view-list.svg
https://github.com/Templarian/MaterialDesign/blob/master/icons/svg/playlist-remove.svg